Assets/Scripts/HealthBar.cs

```diff
@@ -0,0 +1,27 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+public class HealthBar : MonoBehaviour
+{
+    [SerializeField]
+    private Image foregroundImage;
+
+    public void SetMaxHealth(float percent)
+    {
+        foregroundImage.fillAmount = percent;
+    }
+
+    public void SetHealth(float percent)
+    {
+        foregroundImage.fillAmount = percent;
+    }
+
+    private void LateUpdate()
```

Comment:
[TODO] We could create a custom shader that renders the billboard directly and assign it to the GameObjects that need to, so we won't need to rotate the transform in `LateUpdate`
Motivation: More efficient.

Reply:
@RazvanRotaru